test/sengoku_web/channels/game_channel_test.exs

```diff
@@ -1,28 +1,28 @@
 defmodule SengokuWeb.GameChannelTest do
   use SengokuWeb.ChannelCase
 
-  alias SengokuWeb.GameChannel
+  # alias SengokuWeb.GameChannel
```

Comment:
I commented-out the tests in the file, since this functionality is easy to confirm in the browser. It's also likely to change very soon, so I'd rather not write tests to confirm behavior that will change shortly.